float64[21] angleAll